pkg/kubelet/network/dns/dns.go

```diff
@@ -409,7 +409,7 @@ func (c *Configurer) GetPodDNS(pod *v1.Pod) (*runtimeapi.DNSConfig, error) {
 				dnsConfig.Servers = append(dnsConfig.Servers, ip.String())
 			}
 			dnsConfig.Searches = c.generateSearchesForDNSClusterFirst(dnsConfig.Searches, pod)
-			dnsConfig.Options = defaultDNSOptions
+			dnsConfig.Options = appendOptions(defaultDNSOptions, dnsConfig.Options...)
```

Comment:
see related discussion, specially this comment from @bowei here https://github.com/kubernetes/kubernetes/pull/91052#issuecomment-629397532

Reply:
> see related discussion, specially this comment from @bowei here [#91052 (comment)](https://github.com/kubernetes/kubernetes/pull/91052#issuecomment-629397532)

I read it.

But there is still no way to set `options timeout:2 ndots:2` in pod when dnsPolicy=ClusterFirst.
It is set directly to `ndots:5`.

we must spec dncConfig ib every pod to do this.